### ISSUE TITLE
Data normalization before feeding to the Segmentation network

### ISSUE THREAD

**Issue by koutilya-pnvr** [valeoai/ADVENT#26]
Hi, I looked at the dataloader and training codes completely and noticed that you are subtracting the Imagenet mean from the input RGB images (GTA and Cityscapes) but not normalizing the data to either 0 to 1 (or to -0.5 to 0.5 for that matter). 

Just wanted to know if I missed out on something or you intentionally chose to skip the normalization before feeding the data to the segmentation network.

Appreciate your response.

**Comment by soans1994** [comment]
yes, the values are 0-255, i wonder why it has not been divided by 255. 
Also after transformation using mean i get black image is this right? 
![Figure_2](https://user-images.githubusercontent.com/30766648/167596956-4d1ebf41-a156-4a3b-bf88-5499c45489ac.png)
![Figure_1](https://user-images.githubusercontent.com/30766648/167596972-c728be4f-f7cf-49e3-9b4f-9ce7282e092f.png)
